src/generic_ui/scripts/ui.ts

```diff
@@ -119,6 +119,10 @@ export class UserInterface implements ui_constants.UiApi {
   /* About this uProxy installation */
   public availableVersion :string = null;
 
+  /* Platform */
+  private isFirefox :boolean;
+  private isAndroid :boolean;
```

Comment:
Generally we have all browser specific logic in the {chrome,firefox,cordova}_browser_api.ts files.  Could we move this logic there?  We could then set a boolean like ```allowCloudCreation``` to ```false``` in cordova_browser_api.ts and to ```true``` elsewhere